src/magic.cpp

```diff
@@ -1785,6 +1788,22 @@ void spellcasting_callback::draw_spell_info( const spell &sp, const uilist *menu
         }
         damage_string = string_format( "%s %d %s", _( "Summon" ), sp.damage(), _( monster_name ) );
         aoe_string = string_format( "%s: %d", _( "Spell Radius" ), sp.aoe() );
+    } else if( fx == "targeted_polymorph" ) {
+        std::string monster_name = sp.effect_data();
+        if( sp.has_flag( spell_flag::POLYMORPH_GROUP ) ) {
+            // TODO: Get a more user-friendly group name
+            if( MonsterGroupManager::isValidMonsterGroup( mongroup_id( sp.effect_data() ) ) ) {
+                monster_name = "random creature";
```

Comment:
```suggestion
                monster_name = _( "random creature" );
```
And get rid of the translation of `monster_name` at the end. The string "random creature" is most likely not extracted for translation and the game will therefor no know how to translate it anyway. And the result of `monster::get_name()` (called below) is already translated and can not be translated again.